person_class.rb

```diff
@@ -1,8 +1,9 @@
 require './corrector'
+require_relative 'rental'
 # Person class
 class Person
   attr_reader :id
-  attr_accessor :name, :age, :parent_permission
+  attr_accessor :name, :age, :parent_permission, :rental
 
```

Comment:
- [ ] Let's correct the typo error here as we are to add `:rentals` to `attr_accessor` and not `:rental`